layouts/_default/list.md

```
@@ -0,0 +1,33 @@
+{{- /* ABOUTME: Markdown mirror of a section/list page for AI/LLM agents (a14y). */ -}}
+{{- /* ABOUTME: Lists all pages in the section with titles, dates, and URLs. */ -}}
+{{- $description := "" -}}
+{{- with .Description -}}{{ $description = . }}{{- end -}}
+{{- if not $description -}}{{ $description = printf "Index of pages under %q on %s." .Title .Site.Title }}{{- end -}}
+{{- $lastmod := .Lastmod -}}
+{{- if $lastmod.IsZero -}}{{ $lastmod = now }}{{- end -}}
+---
+title: {{ .Title | jsonify }}
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)